### PR TITLE
Setting up the serverless framework

### LINKED ISSUE #39: New serverless backend
Implement a serverless api using Azure functions in nodejs

### PR DESCRIPTION
**Pull request by crysfel**
Setting up the serverless framework, currently there are only three dummy functions and two services.

- hello service, which is a simple hello word service xD
  https://codingcoach-hello.azurewebsites.net/api/hello
- users service, which will be responsible to save and return users/mentors
  https://codingcoach-users.azurewebsites.net/api/list
  https://codingcoach-users.azurewebsites.net/api/store

This is the initial setup to get the ball rolling :)

closes #39

### CODE REVIEW

**Comment by jagzviruz** [comment]
that's a lot of deletions 😱

**Comment by crysfel** [comment]
@jagzviruz unfortunately I didn't see how the previous code would integrate with the serverless framework and azure functions 😅

**Comment by davisokoth** [comment]
@crysfel I think this is great. I also noticed a lot of deletions would be needed.

File: README.md [hunk 1]
```diff
@@ -1,127 +1,54 @@
-# Coding-Coach-api
-[![Build Status](https://api.travis-ci.org/Coding-Coach/coding-coach-api.svg?branch=development)](https://travis-ci.org/Coding-Coach/coding-coach-api)
+# Serverless Azure Functions Node.js Template
 
+This starter template allows quickly creating a Node.js-based service to Azure Functions. It relies on the `serverless-azure-functions` plugin, and therefore, before you can deploy it, you simply need to run `npm install` in order to acquire it (this dependency is already saved in the `package.json` file).
```
Comment (moshfeu) [inline README.md:3]:
```suggestion
This starter template allows quickly creating a Node.js-based service to Azure Functions. It relies on the `serverless-azure-functions` plugin, and therefore, before you can deploy it, you simply need to run `yarn install` in order to acquire it (this dependency is already saved in the `package.json` file).
```
npm or yarn?

**Review by jagzviruz — APPROVED** [review]
Looks good as a starter for Serverless.

**Pull request merged**